js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -937,3 +937,152 @@ cibaIntervalHelp=Die Mindestzeit in Sekunden, die das Endgerät (CD) zwischen Ab
 rolesPermissionsHint=Legt fest, ob feingranulare Berechtigungen für diese Rolle aktiv sein sollen. Wird diese Option deaktiviert, werden alle aktuell vorhandenen feingranularen Berechtigungen gelöscht.
 identityProvidersPermissionsHint=Legt fest, ob feingranulare Berechtigungen für diese Rolle aktiv sein sollen. Wird diese Option deaktiviert, werden alle aktuell vorhandenen feingranularen Berechtigungen gelöscht.
 groupsPermissionsHint=Legt fest, ob feingranulare Berechtigungen für diese Rolle aktiv sein sollen. Wird diese Option deaktiviert, werden alle aktuell vorhandenen feingranularen Berechtigungen gelöscht.
+clientAuthorization=Autorisierung
+path=Pfad
+pairwiseSubAlgorithmSalt.label=Salz
+times.years=Jahre
+organizationsEnabled=Unternehmen
+month=Monat
+dayMonth=Tag
+otpSupportedApplications.totpAppMicrosoftAuthenticatorName=Microsoft Authenticator
+editProvider=Anbieter bearbeiten
+searchProfile=Profil suchen
+renameGroup=Gruppe umbenennen
+used.DEFAULT=Standard
+sslType.all=Alle Anfragen
+validatorDialogColNames.colDescription=Beschreibung
+generateKeys=Schlüssel generieren?
+groupRemove_one=Gruppe entfernt
+group=Gruppe
+iconUri=Icon URI
+idpType.custom=Benutzerdefiniert
+clientScopeType.default=Standard
+valueLabel=Wert
+deleteConfirmTitle_one=Gruppe löschen?
+residentKey.No=Nein
+repeat=Wiederholen
+admin=Admin
+cibaBackhannelTokenDeliveryModes.ping=Ping
+unknownUser=Anonym
+membership=Mitgliedschaft
+uris=URIs
+local=Lokal
+name=Name
+previous=Vorherhige
+prompts.none=Keine
+minus=Minus
+organizationsList=Unternehmen
+hide=Verstecken
+plus=Plus
+policyEnforcementModes.DISABLED=Deaktiviert
+none=Keine
+keyLabel=Schlüssel
+data=Daten
+logicType.positive=Positiv
+spinnerLoading=Laden
+translationsTableHeading=Übersetzungen
+translationTableColumnName=Übersetzung
+defaultLanguage=Standard
+organizations=Unternehmen
+placeholderText=Eines auswählen
+columnName=Name
+protocolTypes.all=Alle
+requirements.ALTERNATIVE=Alternative
+authorization=Authorisierung
+clientScopeSearch.protocol=Protokoll
+organization=Unternehmen
+diagramView=Diagramm-Ansicht
+product=Produkt
+groupDeleted_one=Gruppe gelöscht
+allTypes=Alle Typen
+ldapFilter=LDAP Filter
+editAttribute=Attribut bearbeiten
+showLess=Weniger anzeigen
+deleteConfirmUsers=Benutzer löschen?
+storePassword=Passwort speichern
+searchInitialAccessToken=Token suchen
+continue=Weiter
+unlock=Entsperren
+idpType.social=Social Login
+yes=Ja
+importRole=Rolle importieren
+authenticatorAttachment.platform=Plattform
+sslType.external=Externe Anfragen
+showMetaData=Metadaten anzeigen
+deleteNode=Node löschen?
+addAaguids=AAGUID hinzufügen
+createUser=Benutzer erstellen
+refreshTokens=Tokens aktualisieren
+kc.identity.authc.method=Authentifizierungsmethode
+clientProfiles=Kunden-Profile
```

Comment:
"Client" might again be a untranslated term in the context of Keycloak, as it defines an application that is set up to work with Keycloak.